### PR TITLE
Install multiple

### PR DESCRIPTION
**Pull request by tomwilsonsco**
A tester suggested it would be good to be able to install multiple packages at once with the Addin. In fact they were trying to do this and it wasn't working. This makes changes to allow this - the package names should be separated by commas in the input box and the readme instructions were updated.

I've added some string checking also so it does not proceed if enter anything other than characters, letters, comma - package names can't have any special characters.

### CODE REVIEW

File: R/package_installer.R [hunk 1]
```diff
@@ -11,13 +27,17 @@
...
   pkgs_server <- get_server_path()
+
   pkg_name <- rstudioapi::showPrompt(
-    "Enter Package Name",
+    "Enter Package Name(s)",
     paste(
       "Install from",
       substr(pkgs_server, 6, nchar(pkgs_server))
     )
   )
```
Comment (alice-hannah) [inline R/package_installer.R]: I think I'd expand this to explicitly say multiple packages can be installed by separating with a comma. I know that's probably instinct, but unless people spot the plural 'name(s)' they might not realise they can install multiple.

File: R/package_installer.R [hunk 2]
```diff
@@ -1,7 +1,23 @@
+clean_pkg_str <- function(pkg_str){
```
Comment (alice-hannah) [inline R/package_installer.R]: To fix lintr warning:
```suggestion
clean_pkg_str <- function(pkg_str) {
```

File: R/package_installer.R [hunk 3]
```diff
@@ -1,7 +1,23 @@
...
+  # Remove whitespace
+  pkg_str <- gsub(" ", "", pkg_str)
+
+  # Fail if anything not character, number, comma
+  if (grepl("[^0-9a-z,]", pkg_str, ignore.case=TRUE)){
```
Comment (alice-hannah) [inline R/package_installer.R]:
```suggestion
  if (grepl("[^0-9a-z,]", pkg_str, ignore.case = TRUE)) {
```

**Review by alice-hannah — APPROVED** [review]
Hi Tom - think this is a good idea and works well. Just left a suggestion for the install addin window and to fix a couple of lintr warnings.

Thanks,
Alice

File: R/package_installer.R (hunk 1 above)
Comment (tomwilsonsco) [reply R/package_installer.R]: Thanks I forgot to run `styler::style_pkg()` in my rush to do this before our meetings. Even though we were talking about it earlier :)

**Pull request merged**